src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -59,7 +59,7 @@
     private static final String ISO_8601_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
     private static final int MILLISECONDS_PER_SECOND = 1000;
 
-    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I'm not sure we want to do this - it might be configurable in the future to configure a parser with its own objectmapper instance (e.g. for pretty printing or whatever)